checkpoint_quantum_firewall/assets/logs/checkpoint-quantum-firewall.yaml

```diff
@@ -0,0 +1,946 @@
+    name: Server Inbound Packets
+    path: checkpoint.quantum.firewall.server_inbound_packets
+    source: log
+    type: integer
+  - facetType: range
+    groups:
+      - Checkpoint Quantum Firewall
+    name: Server Outbound Packets
+    path: checkpoint.quantum.firewall.server_outbound_packets
+    source: log
+    type: integer
+  - facetType: list
+    groups:
+      - Checkpoint Quantum Firewall
+    name: Severity
+    path: checkpoint.quantum.firewall.severity
+    source: log
+    type: string
+  - facetType: list
+    groups:
+      - Checkpoint Quantum Firewall
+    name: SmartDefense Profile
+    path: checkpoint.quantum.firewall.smartdefense_profile
+    source: log
+    type: string
+  - facetType: list
+    groups:
+      - Checkpoint Quantum Firewall
+    name: Source
+    path: checkpoint.quantum.firewall.src
+    source: log
+    type: string
+  - facetType: list
+    groups:
+      - Checkpoint Quantum Firewall
+    name: Source Machine Name
+    path: checkpoint.quantum.firewall.src_machine_name
+    source: log
+    type: string
+  - facetType: list
+    groups:
+      - Checkpoint Quantum Firewall
+    name: Status
+    path: checkpoint.quantum.firewall.status
+    source: log
+    type: string
+  - facetType: list
+    groups:
+      - Checkpoint Quantum Firewall
+    name: Subject
+    path: checkpoint.quantum.firewall.subject
+    source: log
+    type: string
+  - facetType: list
+    groups:
+      - Checkpoint Quantum Firewall
+    name: Source IP Port
+    path: checkpoint.quantum.firewall.src_ip_port
+    source: log
+    type: string
+  - facetType: list
+    groups:
+      - Checkpoint Quantum Firewall
+    name: Destination IP Port
+    path: checkpoint.quantum.firewall.dst_ip_port
+    source: log
+    type: string
+  - facetType: range
+    groups:
+      - Checkpoint Quantum Firewall
+    name: Total Bytes
+    path: checkpoint.quantum.firewall.bytes
+    source: log
+    type: integer
+    unit:
+      family: bytes
+      name: byte
+pipeline:
+  type: pipeline
+  name: Checkpoint Quantum Firewall
+  enabled: true
+  filter:
+    query: "source:checkpoint-quantum-firewall"
+  processors:
+    - type: service-remapper
+      name: Define `product` as the official service of the log
+      enabled: true
+      sources:
+        - product
+    - type: date-remapper
+      name: Define `time` as the official date of the log
+      enabled: true
+      sources:
+        - time
+    - name: Lookup for `severity` to `checkpoint.quantum.firewall.severity` field
+      enabled: true
+      source: severity
+      target: checkpoint.quantum.firewall.severity
+      lookupTable: |-
+        0, Info
+        1, Low
+        2, Medium
+        3, High
+        4, Critical
+        Info, Info
+        Low, Low
+        Medium, Medium
+        High, High
+        Critical, Critical
+      type: lookup-processor
+    - type: attribute-remapper
+      name: Map `subject` to `checkpoint.quantum.firewall.subject`
```

Comment:
Given that all logs attributes are now queryable independently of whether they are facets, the main reason to create facets for attributes is to define a unit if the attribute is a measure. Even if in some cases we want to highlight certain integration attributes in the UI, in this case we're creating a very high number of integration specific facets, is there a reason to have that many ?

Reply:
Hi @txominpelu Tagging you in a slack group where this decision was made.